images/Dockerfile.rhel

```diff
@@ -1,10 +1,23 @@
-ARG BASE_IMAGE=rockylinux:9
+ARG BASE_IMAGE
+ARG FLAVOR
 
-FROM $BASE_IMAGE
+FROM $BASE_IMAGE AS base
 
 RUN echo "install_weak_deps=False" >> /etc/dnf/dnf.conf
 
+# Install packages according to distro
+# epel-packages https://docs.fedoraproject.org/en-US/epel/
+FROM base AS rockylinux
 RUN dnf install -y epel-release && dnf clean all
+RUN dnf install -y "https://zfsonlinux.org/epel/zfs-release-2-3$(rpm --eval "%{dist}").noarch.rpm" && dnf clean all
+
+FROM rockylinux AS almalinux
```

Comment:
This breaks my head, can't we name it in a more intuitively way?

Reply:
good point, I added an `epel` stage and both alma and rocky will be based on that one